stylesheets/_typography.scss

```diff
@@ -40,13 +40,13 @@ $is-print: false !default;
   font-weight: $font-weight;
```

Comment:
Having less forks calms my simple mind 👍 

Will let someone else have a peek at this before merging..